Tests/SentryTests/Integrations/SessionReplay/SentrySessionReplayTests.swift

```diff
@@ -251,6 +251,69 @@ class SentrySessionReplayTests: XCTestCase {
         XCTAssertNotNil(fixture.screenshotProvider.lastImageCall)
     }
     
+    func testPauseResume_FullSession() {
+        let fixture = Fixture()
+        
+        let sut = fixture.getSut(options: SentryReplayOptions(sessionSampleRate: 1, onErrorSampleRate: 1))
+        sut.start(rootView: fixture.rootView, fullSession: true)
+        
+        fixture.dateProvider.advance(by: 1)
+        Dynamic(sut).newFrame(nil)
+        XCTAssertNotNil(fixture.screenshotProvider.lastImageCall)
+        sut.pause()
+        fixture.screenshotProvider.lastImageCall = nil
+        
+        fixture.dateProvider.advance(by: 1)
+        Dynamic(sut).newFrame(nil)
+        XCTAssertNil(fixture.screenshotProvider.lastImageCall)
+        
+        fixture.dateProvider.advance(by: 4)
+        Dynamic(sut).newFrame(nil)
+        XCTAssertNil(fixture.replayMaker.lastCallToCreateVideo)
+        
+        sut.resume()
+        
+        fixture.dateProvider.advance(by: 1)
+        Dynamic(sut).newFrame(nil)
+        XCTAssertNotNil(fixture.screenshotProvider.lastImageCall)
+        
+        fixture.dateProvider.advance(by: 5)
+        Dynamic(sut).newFrame(nil)
+        XCTAssertNotNil(fixture.replayMaker.lastCallToCreateVideo)
```

Comment:
`m`: Wouldn't it make sense to assert some actual values of `lastImageCall` and `lastCallToCreateVideo` to ensure everything works correctly? Only asserting it's not nil doesn't say much.

Reply:
I just need to know that nothing was called/not called. The right values are asserted in other tests.